index.html

```diff
@@ -72,7 +72,15 @@ <h2 class="subtitle">
       <div class="container">
         <h2 class="section-title">blog</h2>
         <div class="columns">
-          <div class="column"></div>
+          <div class="column">
+            <TABLE>
```

Comment:
please use lowercase HTML node names, e.g. `<table />`

Reply:
please use the html styles provided by Bulma as base styles. 

What I mean is – use the basic classes as presented here: https://bulma.io/documentation/elements/table/

Then extend with your custom classes and styles